src/robot_gnss/test.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Тест получения данных с GNSS модуля
+# Скрипт копит значения и выдает среднее арифметическое
```

Comment:
Translate all Russian text in English, please.

Reply:
Fixed